Source/Core/Common/PcapFile.h

```diff
@@ -22,9 +22,19 @@
 class PCAP final
 {
 public:
+  enum class Encapsulation : u32
```

Comment:
Wouldn't this be more suited to be named LinkType or DataLinkType, since that's what field in the PCAP header it's used for/representing?

Also the general convention with enum classes in the codebase is to CamelCase them like we regularly do (including uppercasing initialisms): e.g.

```cpp
enum class LinkType : u32
{
    Ethernet,
    User
};
```